.eslintrc.js

```diff
@@ -140,6 +140,13 @@ module.exports = {
       'babel/no-invalid-this': 'off',
       'func-names': 'off',
       'import/order': 'off',
+
```

Comment:
This was in `test/.eslintrc.json`, I moved it here to have better control over it